input/includes/privacy.md

```diff
@@ -0,0 +1,5 @@
+Due to size constraints and to preserve patient privacy, **information that is not _strictly_ necessary for Verifiers in the context of [this IG's use cases](index.html#use-cases) SHALL NOT be included in SMART Health Cards.**
+
+If private or sensitive information are included in a SMART Health Card, it will be exposed to the Verifier every time the SMART Health Card is presented. Additionally, information cannot be removed without invaliding the entire SMART Health Card.[^explanation] It is therefore critical for Issuers to avoid including any information that could represent a safety or privacy risk to a patient in a SMART Health Card.
+
+[^explanation]: SMART Health Cards are designed to be "tamper-evident," meaning that **any** changes to the contents of a SMART Health Card after issuing will be apparent to a Verifier. Typically, Verifiers will reject any SMART Health Cards that have evidence of tampering. For technical details on how tampering can be detected, please refer to [the SMART Health Card specification](https://spec.smarthealth.cards/#issuer-generates-results).
```

Comment:
Suggest saying something about **why** it may be tampered with; e.g., If an issuer includes private information in the SHC, and the holder of the SHC attempts to remove that data to prevent the verifier from seeing the private information, then the holder presents the altered SHC to a verifier (either with the same cryptographic signature, or computing a new signature using a different private key), the tampering would be evident when the verifier checks the SHC.

On one of our calls, someone mentioned https://en.wikipedia.org/wiki/Deadnaming . If an issuer includes such a prior name in the SHC, the holder may not want the verifier to see it. So even something allowed in the DM profile could be used to carry private information.

Reply:
I addressed in acb60a5. Thanks!